### PR TITLE
VerboseLogger should just wrap the parent Logger

### PR DESCRIPTION
**Pull request by Hixie**
This way, you can test things even with a VerboseLogger.

### CODE REVIEW

**Comment by Hixie** [comment]
cc @tvolkert

File: packages/flutter_tools/test/logger_test.dart [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
```
Comment (tvolkert) [inline packages/flutter_tools/test/logger_test.dart]: 2017

Comment (tvolkert) [inline packages/flutter_tools/test/logger_test.dart]: New hotness is to put the test in the same folder structure as the file under test, so this belongs in `test/src/base/logger_test.dart`

File: packages/flutter_tools/lib/src/base/logger.dart [hunk 2]
```diff
@@ -235,13 +237,13 @@ class VerboseLogger extends Logger {
...
+      parent.printError(prefix + terminal.bolden(indentMessage));
       if (stackTrace != null)
-        stderr.writeln(indent + stackTrace.toString().replaceAll('\n', '\n$indent'));
+        parent.printError(indent + stackTrace.toString().replaceAll('\n', '\n$indent'));
     } else if (type == _LogType.status) {
-      print(prefix + terminal.bolden(indentMessage));
+      parent.printStatus(prefix + terminal.bolden(indentMessage));
     } else {
-      print(prefix + indentMessage);
+      parent.printStatus(prefix + indentMessage);
```
Comment (tvolkert) [inline packages/flutter_tools/lib/src/base/logger.dart]: FYI, https://github.com/flutter/flutter/issues/10021 (I didn't have the heart to send you any further down this hole).

**Review by tvolkert — APPROVED** [review]
LGTM

File: packages/flutter_tools/test/logger_test.dart (hunk 1 above)
Comment (Hixie) [inline packages/flutter_tools/test/logger_test.dart]: done

**Comment by Hixie** [comment]
will check in on green

**Pull request merged**